Core/MoneyConverter.cs

```diff
@@ -12,5 +12,17 @@ public static Money Convert(this Money money, Currency currency)
             }
             throw new ArgumentException("Invalid Currency");
         }
+
+        public static Currency Convert(string currency)
+        {
+            switch (currency.ToLower())
+            {
+                case "uah":
+                case "грн":
+                    return Currency.UAH;
+                default:
+                    throw new ArgumentException();
```

Comment:
Codacy found an issue: [Use a constructor overloads that allows a more meaningful exception message to be provided.](https://app.codacy.com/manual/siriak/MoneyManager/pullRequest?prid=5924375)